_opensearch/install/tar.md

```diff
@@ -7,144 +7,455 @@ nav_order: 50
+   openssl genrsa -out root-ca-key.pem 2048
+   
+   # Use the private key to create a self-signed root certificate. Be sure to
+   # replace the arguments passed to -subj so they reflect your specific host.
+   openssl req -new -x509 -sha256 -key root-ca-key.pem -subj "/C=CA/ST=ONTARIO/L=TORONTO/O=ORG/OU=UNIT/CN=ROOT" -out root-ca.pem -days 730
+   ```
+1. Next, create the admin certificate. This certificate is used is used to gain elevated rights for performing administrative tasks relating to the security plugin.
+   ```bash
+   # Create a private key for the admin cert
+   openssl genrsa -out admin-key-temp.pem 2048
+
+   # Convert the private key to PKCS#8
+   openssl pkcs8 -inform PEM -outform PEM -in admin-key-temp.pem -topk8 -nocrypt -v1 PBE-SHA1-3DES -out admin-key.pem
+   
+   # Create the CSR. A common name (CN) of "A" is acceptable because this certificate is
+   # used for authenticating elevated access and is not tied to a host.
+   openssl req -new -key admin-key.pem -subj "/C=CA/ST=ONTARIO/L=TORONTO/O=ORG/OU=UNIT/CN=A" -out admin.csr
+   
+   # Sign the admin certificate with the root certificate and private key that was created earlier
+   openssl x509 -req -in admin.csr -CA root-ca.pem -CAkey root-ca-key.pem -CAcreateserial -sha256 -out admin.pem -days 730
+   ```
+1. Create a certificate for the node being configured.
+   ```bash
+   # Create a private key for the node cert
+   openssl genrsa -out node1-key-temp.pem 2048
+   
+   # Convert the private key to PKCS#8
+   openssl pkcs8 -inform PEM -outform PEM -in node1-key-temp.pem -topk8 -nocrypt -v1 PBE-SHA1-3DES -out node1-key.pem
+   
+   # Create the CSR - replace the arguments passed to -subj to they reflect your specific host.
+   # The CN should match a DNS A record for the host - do not use the hostname.
+   openssl req -new -key node1-key.pem -subj "/C=CA/ST=ONTARIO/L=TORONTO/O=ORG/OU=UNIT/CN=node1.dns.a-record" -out node1.csr
+   
+   # Create an extension file that defines a SAN DNS name for the host. This
+   # should match the DNS A record of the host.
+   echo 'subjectAltName=DNS:node1.dns.a-record' > node1.ext
+   
+   # Sign the node certificate with the root certificate and private key that was created earlier
+   openssl x509 -req -in node1.csr -CA root-ca.pem -CAkey root-ca-key.pem -CAcreateserial -sha256 -out node1.pem -days 730 -extfile node1.ext
+   ```
+1. Remove temporary files that are no longer required.
+   ```bash
+   rm *temp.pem *csr *ext
+   ```
+1. Add these certificates to `opensearch.yml` as described in [Generate Certificates]({{site.url}}{{site.baseurl}}/security-plugin/configuration/generate-certificates/#add-distinguished-names-to-opensearchyml). Advanced users might also choose to append the settings using a script:
+   ```bash
+   #! /bin/bash
+
+   # Before running this script, make sure to replace the /path/to your OpenSearch directory
+   # and don't forget to replace the CN in the node's distinguished name with a real
```

Comment:
Instead of "don't forget to replace" can we say "...and remember to replace the CN..."